.github/workflows/ci.yml

```diff
@@ -450,10 +454,12 @@ jobs:
     if: github.event_name == 'release' && github.event.action == 'published'
     strategy:
       matrix:
-        suite: [jammy]
+        suite: [jammy, bionic]
         include:
           - suite: jammy
-            os-version: ubuntu-22.04
+            os-version: ubuntu-20.04
+          - suite: bionic
+            os-version: ubuntu-18.04
       fail-fast: false
     runs-on: ${{ matrix.os-version }}
```

Comment:
So, basically this would need to become `ubuntu-22.04` ?

Reply:
Yes, all GHA runner can be on ubuntu-22.04, however, bionic is EOL and bullseye and bookworm should be supported, i.e. packages built and published.

Reply:
Okay, could you please try to switch this? I'd push the commit myself, but this PR is from your fork, meaning I can't easily do it.

Reply:
Did some changes to Debian build and publish actions, however, the release and publish parts won't be triggered on a PR.